blueocean-rest/src/main/java/io/jenkins/blueocean/rest/ApiHead.java

```diff
@@ -50,11 +52,7 @@ public ApiHead() {
      */
     @WebMethod(name="search") @GET @PagedResponse
     public Pageable<?> search(@QueryParameter("q") Query query) {
-        for (OmniSearch os : OmniSearch.all()) {
-            if (os.getType().equals(query.type))
-                return os.search(query);
-        }
-        return Pageables.empty();
+        return OmniSearch.query(query, Jenkins.getInstance());
```

Comment:
@kzantow Sorry, Same as other comment, we do not want to define Jenkins dependency directly on blueocean-rest module 🐛